### ISSUE TITLE
Failed to remove directory when running /remove_cache and cache not cleared

### ISSUE THREAD

**Issue by GhaziTriki** [forkcms/forkcms#1698]
## Type

- Non critical bug

## Problem description

Three errors are displayed in console when running `remove_cache`in tools. The cache directories are not deleted.

The line 26 of remove_cache only works when run as sudo.

![image](https://cloud.githubusercontent.com/assets/4991088/18007724/856b53b8-6ba5-11e6-922a-ab5d17bffba0.png)

## Steps to reproduce

Just run the following command `./tools/remove_cache`

## Expected behavior / Proposed solutions

Cache folders are deleted using `cache:clear` option.

**Comment by carakas** [comment]
it looks to me like a permission problem on the filesystem

**Comment by GhaziTriki** [comment]
There is no permission issue as you can see in the attached output of `ls -laR app/cache/`. The folder is under a shared vagratn folder with default permissions.
[permissions.txt](https://github.com/forkcms/forkcms/files/439426/permissions.txt)

**Comment by WouterSioen** [comment]
@GhaziTriki does it work when you execute the command from inside your vagrant box/docker container?

**Comment by GhaziTriki** [comment]
Yes it happens when executed from inside a vagrant box, even within a shared folder.

**Comment by tijsverkoyen** [comment]
Can you run `whoami` in your console.

**Comment by GhaziTriki** [comment]
The current user is `vagrant`

**Comment by tijsverkoyen** [comment]
The problem is that de folder is owned by the user `ubuntu` if the permissions.txt is still relevant. And therefore the folders can't be deleted.

**Comment by GhaziTriki** [comment]
I will try changing the ownership of the folder and get back with a feedback.

**Comment by carakas** [comment]
@GhaziTriki any update on this?
